examples/feature_store/README.md

```diff
@@ -0,0 +1,141 @@
+# 🗂 Get your data from a Feature Store
+
+<!-- basic introduction paragraph
+
+- what is a feature store
+- what problem does it solve
+- what specific combo have we integrated with? -->
+
+## 🗺 Overview: Features Stores & ZenML
+
+<!-- - how to think about a feature store
+- what is it used for?
+- components / architecture of a feature store
+  - offline / batch serving
+  - online serving
+- The main abstractions you need to think of / how do we help you?:
+  - getting offline features
+  - getting online features
+- caveats about our integration
+  - this example runs locally
+  - we assume you have a feature store already if you want to use it in
+    production (ZenML doesn't currently help you set that all up)
+  - online serving doesn't work in tandem with deployed models currently
+- This example, and what features of features it showcases / what you can do
+  with the ZenML feature store integration -->
+
+## 🧰 How the example is implemented
+
+<!-- Showcase the code and explain how it works
```

Comment:
Not sure about these comments. Do you intend to take them out?

Reply:
I'll add this this morning.